back/middleware/authMiddleware.ts

```
@@ -0,0 +1,32 @@
+  id: number;
+  email: string;
+  nickname: string;
+}
+
+export const verifyToken = (
+  req: Request & { user?: JwtPayload },
+  res: Response,
+  next: NextFunction
+) => {
+  const authHeader = req.headers['authorization'];
+  const token = authHeader && authHeader.split(' ')[1];
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)